PlayCover/Views/App Views/StoreAppView.swift

```diff
@@ -78,14 +76,14 @@ struct StoreAppView: View {
                     .appendingPathExtension("ipa")
                 try FileManager.default.moveItem(at: url, to: tmpDir)
                 uif.ipaUrl = tmpDir
-                DispatchQueue.main.async {
```

Comment:
This is still reverting the changes done in #449.

Reply:
wtf i did clean fork :D i feel like an idiot rn .